remill/BC/DeadStoreEliminator.cpp

```diff
@@ -1172,7 +1185,7 @@ VisitResult ForwardAliasVisitor::visitCallInst(llvm::CallInst &inst) {
 }
 
 VisitResult ForwardAliasVisitor::visitInvokeInst(llvm::InvokeInst &inst) {
-  auto val = inst.getCalledValue()->stripPointerCasts();
+  auto val = inst.getCalledFunction()->stripPointerCasts();
```

Comment:
`getCalledFunction` might return `nullptr`

Reply:
updated to getCalledOperand